lib/CompatibilityPlugin.js

```diff
@@ -9,6 +9,11 @@ const ConstDependency = require("./dependencies/ConstDependency");
 const NullFactory = require("./NullFactory");
 
 class CompatibilityPlugin {
+	/**
+	 * Apply the plugin
+	 * @param {TODO} compiler Webpack Compiler
```

Comment:
TODO => Compiler

Reply:
I've copied the typing from [mohsens1 typing for DefinePlugin].(https://github.com/webpack/webpack/pull/7356/commits/800e0de25b8e0e0fc9ae69b9e76dbec00fb5895f).
I thought, he has his reasons not to use `Compiler` here.
Np, I will change it.

Reply:
done